snc-library.sh

```diff
@@ -66,13 +66,9 @@ function run_preflight_checks() {
         echo "Checking libvirt and DNS configuration"
 
 	LIBVIRT_URI=qemu:///system
-	if [ ${bundle_type} == "snc" ] || [ ${bundle_type} == "okd" ]; then
-           LIBVIRT_URI=qemu+tcp://localhost/system
-	fi
-
         # check if we can connect to ${LIBVIRT_URI}
         if ! sudo virsh -c ${LIBVIRT_URI} uri >/dev/null; then
-              preflight_failure  "libvirtd is not listening for ${LIBVIRT_URI}, see https://github.com/openshift/installer/tree/master/docs/dev/libvirt#configure-libvirt-to-accept-tcp-connections"
+              preflight_failure  "libvirtd is not listening for ${LIBVIRT_URI}, check if libvirt daemon is running https://libvirt.org/daemons.html"
```

Comment:
I'd phrase this as "libvirtd is not accessible over qemu:///system, check if the libvirt daemon is running"

Reply:
@cfergeau you don't want to have link in the failure message?

Reply:
I did not c&p it, but fine for me with or without, it's the beginning I wanted to rephrase